upgrade/UPGRADE-v8.1.0.md

```diff
@@ -492,6 +556,7 @@ There you can find links to upgrade notes for other versions too.
 - improve functional and smoke tests to be more readable and easier to write [#1392](https://github.com/shopsys/shopsys/pull/1392)
```

Comment:
```suggestion
- improve functional and smoke tests to be more readable and easier to write ([#1392](https://github.com/shopsys/shopsys/pull/1392))
```

Reply:
Same here. I see no space at the beggining.